### PR TITLE
Cirrus: Show names/versions of critical packages

### PR DESCRIPTION
**Pull request by cevich**
Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

File: contrib/cirrus/logcollector.sh [hunk 1]
```diff
@@ -32,5 +32,42 @@ case $1 in
     df) showrun df -lhTx tmpfs ;;
     ginkgo) showrun cat $CIRRUS_WORKING_DIR/test/e2e/ginkgo-node-*.log ;;
     journal) showrun journalctl -b ;;
+    packages)
+        case $OS_RELEASE_ID in
+            fedora*)
+                PKG_LST_CMD='rpm -q --qf=%{N}-%{V}-%{R}-%{ARCH}\n'
+                PKG_NAMES=(\
+                    container-selinux \
```
Comment (rhatdan) [inline contrib/cirrus/logcollector.sh]: How come the list of packages is different between fedora and ubuntu?

Comment (cevich) [reply contrib/cirrus/logcollector.sh]: Totally different packaging systems for one, the Ubuntu packages come from the PPA that @lsm5 robots maintain.  Though TBH the lists here are somewhat arbitrary...I grabbed them from `contrib/cirrus/packer/{fedora,ubuntu}_setup.sh`.  Happy to add/change/remove packages to log here, or in a followup PR.  Any suggestions?

**Comment by cevich** [comment]
The new output shows up in the CirrusCI WebUI:

screenshot

**Comment by mheon** [comment]
/approve
LGTM

**Pull request merged**